tests/page_test.go

```diff
@@ -961,7 +960,7 @@ func TestPageOn(t *testing.T) { //nolint:gocognit
 			assertFn: func(t *testing.T, cm *common.ConsoleMessage) {
 				t.Helper()
 				assert.Equal(t, "timeEnd", cm.Type)
-				assert.True(t, strings.HasPrefix(cm.Text, "k6: 0."), `missing expected prefix "k6: 0."`)
+				assert.Regexp(t, "^k6: [0-9]*.[0-9]*", cm.Text, `expected prefix "k6: 0." but got %q`, cm.Text)
```

Comment:
Can you help me understand why we use a regex here?

Reply:
When working with `console.time` it calculates the time in between the two calls `console.time('k6');` and  `console.timeEnd('k6');` and prints the delta in the console output. Since it's time we can't be sure how long it will take. In CI it seems to be quite a bit slower than running it locally. Using a regex to verify that the prefix of the output is made up of `k6` and a number.

Reply:
Thanks for explaining. So, why does the assert error message say `expected prefix "k6: 0."`? Since we try to match for k6: `0-9.0-9` instead of `k6: 0.0-9`.

Reply:
Good spot, i've changed it in f604d5807fe828169b67482d2e9c24ab9238fdd1